src/sentry_user.h

```diff
@@ -0,0 +1,43 @@
+#ifndef SENTRY_USER_H
+#define SENTRY_USER_H
+
+#include <godot_cpp/classes/ref_counted.hpp>
+#include <godot_cpp/variant/string.hpp>
+
+using namespace godot;
+
+class SentryUser : public RefCounted {
+	GDCLASS(SentryUser, RefCounted);
+
+private:
+	String user_id;
+	String username;
+	String email;
+	String ip_address;
+
+protected:
+	static void _bind_methods();
+
+	String _to_string() const;
+
+public:
+	void set_user_id(const String &p_user_id) { user_id = p_user_id; }
+	String get_user_id() const { return user_id; }
+
+	void set_username(const String &p_username) { username = p_username; }
+	String get_username() const { return username; }
+
+	void set_email(const String &p_email) { email = p_email; }
+	String get_email() const { return email; }
+
+	void set_ip_address(const String &p_ip_address) { ip_address = p_ip_address; }
+	String get_ip_address() const { return ip_address; }
+
+	void infer_ip_address() { ip_address = "{{auto}}"; }
```

Comment:
Godot has a built-in documentation system. I think, explanations would need to go there, because that's where most users would search for answers. 
![image](https://github.com/user-attachments/assets/f616c6ef-a2b3-4a55-8b5e-3f2a8404f4a5)
I'll tackle built-in docs in a separate PR, since it needs some code to be added in extension.

Reply:
Got it. Would be cool if that tool pulled docs from code. So code and docs could live at the same location

Reply:
That tool is part of Godot, e.g. `godot --doctool`. It generates XML files for API exposed to `ClassDB`, and also updates those files on subsequent runs, but it only deals with the structure. The descriptions need to be added manually to those XML files. It preserves the added content on subsequent runs. There is also a tool to export sphinx-based RST-formatted class documentation.